rules/gluster-utilization.libsonnet

```diff
@@ -0,0 +1,19 @@
+{
+  prometheusRules+:: {
+    groups+: [
+      {
+        name: 'gluster_volume_capacity.rules',
+        rules: [
+          {
+            expr: |||
+                  'sum(max(gluster_subvol_capacity_total_bytes{%(glusterExporterSelector)}))
```

Comment:
Shouldn't ` max(gluster_subvol_capacity_total_bytes{job="glusterd2-client"})by(volume) ` be sufficient ?

Reply:
I think comment https://github.com/gluster/gluster-mixins/pull/14#discussion_r242515502 explains this why `max` is required.